tests/ec2.rs

```diff
@@ -23,3 +24,47 @@ fn main() {
     }
 
 }
+
+#[test]
+// Test for issue 383
+fn dry_run() {
+    let credentials = DefaultCredentialsProvider::new().unwrap();
+    let ec2 = Ec2Client::new(credentials, Region::UsEast1);
+    let req = CreateSnapshotRequest {
+                     volume_id: "v-00000001".into(),
+                     description: None,
+                     dry_run: Some(true)
+                 };
+    match ec2.create_snapshot(&req) {
+        Ok(_) => {
+            panic!("create_snapshot should fail");
```

Comment:
It worked when  I tested it and that is what the existing test used. But I like not having to match against a case that should not occur and instead just expecting the panic. I'll try using should_panic.